packages/loot-core/src/client/budgets/budgetsSlice.ts

```
@@ -0,0 +1,589 @@
+      await dispatch(loadPrefs());
+    }
+
+    await dispatch(setAppState({ loadingText: null }));
+  },
+);
+
+export const closeBudget = createAppAsyncThunk(
+  `${sliceName}/closeBudget`,
+  async (_, { dispatch, getState }) => {
+    const prefs = getState().prefs.local;
+    if (prefs && prefs.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)